CMakeLists.txt

```diff
@@ -46,22 +46,32 @@ set(TRITON_COMMON_REPO_TAG ${TRITON_BACKEND_API_VERSION} CACHE STRING "Tag for t
 set(TRITON_CORE_REPO_TAG ${TRITON_BACKEND_API_VERSION} CACHE STRING "Tag for triton-inference-server/core repo")
 set(TRITON_BACKEND_REPO_TAG ${TRITON_BACKEND_API_VERSION} CACHE STRING "Tag for triton-inference-server/backend repo")
 
-set(DALI_VERSION "" CACHE STRING
-        "DALI version that should be downloaded by the build system.
-         By default the latest available DALI version will be downloaded.
-         Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON."
-)
-set(DALI_EXTRA_INDEX_URL "https://developer.download.nvidia.com/compute/redist" CACHE STRING
-        "URL of the Index, from which DALI will be downloaded by the build system.
-         Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON."
-)
+set(DALI_DOWNLOAD_VERSION "" CACHE STRING
+    "DALI version that should be downloaded by the build system.
+    By default the latest available DALI version will be downloaded.
+    Ignored, when TRITON_DALI_SKIP_DOWNLOAD is ON.")
+
+set(DALI_DOWNLOAD_NIGHTLY OFF CACHE STRING
+    "Set to ON if the DALI nightly version should be downloaded.
+    Ignored, when TRITON_DALI_SKIP_DOWNLOAD in ON")
+
+if (NOT ${DALI_DOWNLOAD_NIGHTLY})
```

Comment:
How about weekly?

Reply:
Good point. Maybe it's better to change the option to DALI_DOWNLOAD_TYPE with options `release`, `weekly`, `nightly`

Reply:
done